### ISSUE TITLE
Start new chat messages at the top of the screen?

### ISSUE THREAD

**Issue by lpbas** [flyerhq/flutter_firebase_chat_core#116]
Thank you for this awesome library!
I wanted to ask you if there is a way to start the first few messages of a chat, where the whole screen isn't yet filled, from the top of the screen and not from the bottom as in the current chat widget implementation.

If it's not yet possible I'd be happy to look into it and create a PR when I have some time.

**Comment by demchenkoalex** [comment]
Hi! this is not the issue with this package, but with the flutter chat UI. Unfortunately, due to architecture of the current implementation it will be impossible to do it, however I am _very slowly_ working on greatly improved v2 where this will be a default behaviour. Have practically no time with my current workload since start of the summer though, waiting for some time I could get to finish this.

https://github.com/flyerhq/flutter_firebase_chat_core/assets/14123304/454f3cb2-e7f5-477c-a4a0-a2b9c86f8558

**Comment by lpbas** [comment]
You are right about the UI, thank you for clarifying. Looking forward the v2 implementation, it looks great! Thank you for all your hard work @demchenkoalex!
